ftest.sh

```diff
@@ -188,7 +217,8 @@ shift || true
 args+=" $*"
 
 # shellcheck disable=SC2029
-if ! ssh -i ci_key jenkins@"${nodes[0]}" "set -ex
+# shellcheck disable=SC2086
+if ! ssh $SSH_KEY_ARGS "${REMOTE_ACCT:-jenkins}"@"${nodes[0]}" "set -ex
```

Comment:
(lint) The double quotes around this do nothing. Remove or escape them. [SC2140]